### PR TITLE
feat: firestore custom table names

### PR DESCRIPTION
**Pull request by marrrcin**
Readme for #47

### CODE REVIEW

**Comment by marrrcin** [comment]
BTW, @aguacongas what is the process of releasing the new version (with table names feature) to NuGet?

**Comment by aguacongas** [comment]
I 1st create a preview by creating a preview/3.3.0, AppVeyor'll deployed packages to nuget, so you can test those packages.
Then I'll create a release branch release/3.3.0 and AppVeyor'll published the release packages to nuget.

File: src/Aguacongas.Identity.Firestore/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Aguacongas.Identity.Firestore
+Google Firestore-based ASP.NET Core Identity user & role stores.
+
+## Basic usage
+1. Set `GOOGLE_APPLICATION_CREDENTIALS` environment variable to a path to the JSON file for GCP service account (see https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable)
+1. In `Startup.cs` of your ASP.NET Core project:
```
Comment (aguacongas) [inline src/Aguacongas.Identity.Firestore/README.md:6]: duplicate 1.

File: src/Aguacongas.Identity.Firestore/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Aguacongas.Identity.Firestore
+Google Firestore-based ASP.NET Core Identity user & role stores.
+
+## Basic usage
+1. Set `GOOGLE_APPLICATION_CREDENTIALS` environment variable to a path to the JSON file for GCP service account (see https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable)
```
Comment (aguacongas) [inline src/Aguacongas.Identity.Firestore/README.md:5]: Use md link : `[setting_the_environment_variable](https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable)`

File: src/Aguacongas.Identity.Firestore/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+   {
+      // ... your other services 
+       services
+       .AddIdentity<ApplicationUser, IdentityRole>()
+       .AddFirestoreStores("<YOUR GCP PROJECT ID>")
+       .AddDefaultTokenProviders();
+   
+      // ... rest of your configuration
+   }
+    ```
```
Comment (aguacongas) [inline src/Aguacongas.Identity.Firestore/README.md:18]: Remove leading space

**Review by aguacongas — CHANGES_REQUESTED** [review]

File: src/Aguacongas.Identity.Firestore/README.md (hunk 1 above)
Comment (marrrcin) [reply src/Aguacongas.Identity.Firestore/README.md:6]: Markdown automatically numerates the lists:
https://www.markdownguide.org/basic-syntax/#ordered-lists

File: src/Aguacongas.Identity.Firestore/README.md (hunk 2 above)
Comment (marrrcin) [reply src/Aguacongas.Identity.Firestore/README.md]: Done

File: src/Aguacongas.Identity.Firestore/README.md (hunk 3 above)
Comment (marrrcin) [reply src/Aguacongas.Identity.Firestore/README.md:18]: Done.

File: src/Aguacongas.Identity.Firestore/README.md (hunk 1 above)
Comment (aguacongas) [reply src/Aguacongas.Identity.Firestore/README.md:6]: Maybe, but it's ugly. Don't you think ?

**Review by aguacongas — CHANGES_REQUESTED** [review]

File: src/Aguacongas.Identity.Firestore/README.md (hunk 1 above)
Comment (marrrcin) [reply src/Aguacongas.Identity.Firestore/README.md]: I usually don't read raw markdowns :D 
Changed as per request

**Comment by marrrcin** [comment]
Could you release the preview today?

**Comment by aguacongas** [comment]
@marrrcin the preview is deployed to nuget https://www.nuget.org/packages/Aguacongas.Identity.Firestore/3.3.0-preview0058

**Pull request merged**